src/components/MemberContribution/ContributionAccordian.tsx

```diff
@@ -40,14 +40,17 @@ export default function ContributionAccordion({
   const renderData = contribution?.map((data: any, idx: number) => {
     const task = setData(data);
     // title exist boolean
-    const title = !!data?.task?.title;
+    const isTitle =
```

Comment:
Does this variable track if something is title or other element(eg. paragraph)? If not then naming can be improved.

Reply:
Yes it track the title of the task that's why i have name it isTitle